src/screens/InputUsername/index.js

```diff
@@ -162,7 +162,7 @@ const ChooseUsername = () => {
   const next = () => {
     if (username && username.length >= MINIMUM_USERNAME_LENGTH && typeFetch === 'available') {
       setUsername(username, dispatch);
-      navigation.navigate('LocalCommunity');
+      navigation.navigate('Topics');
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The navigation destination has been changed from 'LocalCommunity' to 'Topics'. Ensure that the 'Topics' screen is prepared to handle users coming directly from this point in the onboarding process, and that all necessary data is available.
```diff
-      navigation.navigate('LocalCommunity');
+      navigation.navigate('Topics');
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->